db/R/dbfiles.R

```diff
@@ -39,31 +39,31 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
   
   # find mimetype, if it does not exist, it will create one
   mimetypeid <- get.id("mimetypes", "type_string", mimetype, con, create=TRUE)
-  
+ 
   # find appropriate format, create if it does not exist
```

Comment:
I'm not sure what @robkooper is referring to, as the only difference in this function is whitespace. That said, some of the changes lower down look like ones @tonygardella made, which suggests you might be out of sync with the mainline. I'd recommend a pull upstream master.